src/main/kotlin/com/coder/toolbox/settings/CoderSettings.kt

```diff
@@ -334,14 +334,14 @@ open class CoderSettings(
     val dataDir: Path
         get() {
             return when (getOS()) {
-                OS.WINDOWS -> Paths.get(env.get("LOCALAPPDATA"), "coder-gateway")
-                OS.MAC -> Paths.get(env.get("HOME"), "Library/Application Support/coder-gateway")
+                OS.WINDOWS -> Paths.get(env.get("LOCALAPPDATA"), "coder-toolbox")
+                OS.MAC -> Paths.get(env.get("HOME"), "Library/Application Support/coder-toolbox")
                 else -> {
                     val dir = env.get("XDG_DATA_HOME")
                     if (dir.isNotBlank()) {
-                        return Paths.get(dir, "coder-gateway")
+                        return Paths.get(dir, "coder-toolbox")
```

Comment:
I think we should drop the toolbox suffix here. There is nothing like a Coder toolbox. The coder is part of the JetBrains toolbox.

Reply:
We can probably do it as long as we're sure we are not overlapping (now or in the future) with coder cli manually installed.

Reply:
Hmm. I will then keep the prefix. I was not sure what it was for.
@bcpeinhardt and @f0ssel may have more context here,